tests/lib/rules/comma-dangle.js

```diff
@@ -392,8 +392,8 @@ ruleTester.run("comma-dangle", rule, {
                 {
                     message: "Missing trailing comma.",
                     type: "Property",
-                    line: 2,
-                    column: 11
+                    line: 3,
+                    column: 1
```

Comment:
I don't know if I agree with changing the reported location for these test cases. I think it makes more sense from a user perspective that the reported location is right after the property instead of at the closing brace. 
An example is the test case marked by this comment:
Before:
```bash
  1 | var foo = {
> 2 | bar: 'baz'
    |           ^
  3 | }
```
After:
```bash
  1 | var foo = {
  2 | bar: 'baz'
> 3 | }
    | ^
```
I think the original reported location in this case makes more logical sense for the user, since that's where he will have to insert the trailing comma.

What if you would report on `penultimateToken.loc.end` instead of on `trailingToken.loc.start`? I think that would fix the edge case of this bug report and preserve the location reported on all the other tests cases.